src/app/pages/5_Find_Assay_Target_Area.py

```diff
@@ -63,7 +63,7 @@
         reference_sequence = st.text_input(
             "Select Reference Sequence",
             # [file.name for file in target_files],
-            help="If no selection is made, the first file will be used as the reference sequence",
+            help="Please input filename that will be used as the reference sequence",
```

Comment:
[nit / grammar] "Please input filename that" to "Please input the filename that"

Reply:
I am really bad at grammar, so this may be incorrect.